src/org/rascalmpl/util/maven/Artifact.java

```diff
@@ -151,6 +150,14 @@ private void calculateClassPath(Scope forScope, HashSet<WithoutVersion> alreadyI
         }
     }
 
+    private Artifact createSystemArtifact(Dependency d) {
+        var path = d.getSystemPath();
+        if (path == null) {
```

Comment:
what if the file doesn't exist?

Reply:
Maven already generates a warning in that case.